usaspending_api/common/api_request_utils.py

```diff
@@ -253,6 +253,13 @@ def validate_post_request(self, request):
                                 raise InvalidParameterException("Invalid field, operation 'range_intersect' requires an array of length 2 for field")
                             if (not isinstance(filt['value'], list) or len(filt['value']) != 2) and 'value_format' not in filt:
                                 raise InvalidParameterException("Invalid value, operation 'range_intersect' requires an array value of length 2, or a single value with value_format set to a ranged format (such as fy)")
+                        if filt['operation'] == 'search':
```

Comment:
Yes, `search` has the option of specifying a single field or an array, so we check to see if all the fields are text.